### PR TITLE
Make setup instructions a bit more clear

### PR DESCRIPTION
**Pull request by gregrickaby**
Break out setup into two sections 1) Docker 2) WordPress. Make it clear that Docker needs to be running in order to run setup script.

### CODE REVIEW

**Comment by tlovett1** [comment]
I think this is a bit confusing. Running `docker-compose up` isn't really setting up Docker, rather it's just starting containers.

**Comment by gregrickaby** [comment]
`docker-compose up` is in the original [README](https://github.com/10up/wp-local-docker/blob/master/readme.md) 

My PR is suggesting breaking setup into two parts...

**Comment by gregrickaby** [comment]
@tlovett1 How about this?

File: readme.md [hunk 1]
```diff
@@ -17,14 +17,17 @@ The `/config/elasticsearch/plugins` folder is mapped to the plugins folder in th
...
 1. `cd <my-project-name>`
 1. `docker-compose up`
-1. Run setup to download WordPress and create a `wp-config.php` file.
-	1. On Linux / Unix / OSX, run `sh bin/setup.sh`.
-	2. On Windows, run `./bin/setup`.
+
+## Setup WordPress
+You will need to run a setup script to download WordPress and create a `wp-config.php` file. 
+1. With Docker running, open a new terminal window and CD into `<my-project-name>`
+	1. On Linux / Unix / OSX, run ```bash sh bin/setup.sh```
```
Comment (schoenwaldnils) [inline readme.md:29]: `bash sh bin/setup.sh` gets me this: `/bin/sh: /bin/sh: cannot execute binary file`

**Comment by indietravel** [comment]
I would recommend also changing Setup item three from `3. docker-compose up` to `3. docker-compose up -d` to get around this issue with ElasticPress hanging: https://github.com/elastic/elasticsearch/issues/11567

**Pull request closed without merging**